src/Common/src/Interop/Windows/winhttp/Interop.winhttp.cs

```diff
@@ -28,6 +28,13 @@ internal partial class WinHttp
             ushort serverPort,
             uint reserved);
 
+        [DllImport(Interop.Libraries.WinHttp, EntryPoint = "WinHttpConnect", CharSet = CharSet.Unicode, SetLastError = true)]
+        public static extern SafeWinHttpHandleWithCallback WinHttpConnectWithCallback(
+            SafeWinHttpHandle sessionHandle,
+            string serverName,
+            ushort serverPort,
+            uint reserved);
```

Comment:
Could be worth a comment what these *WithCallback imports are, as other than the name (which is then addressed with an EntryPoint) they look the same as their non-*WithCallback counterparts.